src/PSRule.Rules.Azure/rules/Azure.VM.Rule.ps1

```diff
@@ -279,6 +279,20 @@ Rule 'Azure.VM.SQLServerDisk' -Ref 'AZR-000324' -Type 'Microsoft.Compute/virtual
 
 #endregion IaaS SQL Server disks
 
+#region Azure Monitor Agent
+
+# Synopsis: Use Azure Monitor Agent for collecting monitoring data.
+Rule 'Azure.VM.AMA' -Ref 'AZR-000344' -Type 'Microsoft.Compute/virtualMachines' -Tag @{ release = 'Preview'; ruleSet = '2022_12'; } {
```

Comment:
In the context of general monitoring AMA is generally available so we don't need to mark it as preview.

```suggestion
Rule 'Azure.VM.AMA' -Ref 'AZR-000344' -Type 'Microsoft.Compute/virtualMachines' -Tag @{ release = 'GA'; ruleSet = '2022_12'; } {
```